.release-it.json

```diff
@@ -1,20 +1,23 @@
 {
-  "src": {
-    "tagName": "v%s",
-    "tagAnnotation": "Release v%s",
-    "commitMessage": "chore(Release): v%s",
-    "beforeStartCommand": "rm -rf packages/blockchain-wallet-v4-frontend/build; yarn manage:translations"
+  "scripts": {
+    "beforeStart": [
+      "rm -rf packages/blockchain-wallet-v4-frontend/build",
+      "yarn manage:translations"
+    ]
   },
   "git": {
+    "requireCleanWorkingDir": false,
+    "requireUpstream": false,
+    "_changelog": "changelog -t $(git describe --abbrev=0 --tags) -f -",
+    "commitMessage": "chore(Release): v${version}",
+    "tagName": "v${version}",
+    "tagAnnotation": "Release v${version}",
     "pushArgs": "--no-verify"
   },
   "github": {
     "release": true,
     "preRelease": true,
```

Comment:
@webpro I don't think preRelease is working, or maybe the flag was changed? It creates the `Latest Release` tag, but these are all pre releases, I change to `Latest Release` when we deploy to prod.